editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/preferences/ProjectTemplatesPreferencesConstants.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package org.fusesource.ide.projecttemplates.preferences;
+
+/**
+ * @author bfitzpat
+ */
+public interface ProjectTemplatesPreferencesConstants {
```

Comment:
it is not impacting only project templates, so we might rename it to something like StagingRepositoriesConstants